iocore/net/UnixNet.cc

```diff
@@ -43,6 +43,25 @@ NetHandler::Config NetHandler::global_config;
 std::bitset<std::numeric_limits<unsigned int>::digits> NetHandler::active_thread_types;
 const std::bitset<NetHandler::CONFIG_ITEM_COUNT> NetHandler::config_value_affects_per_thread_value{0x3};
 
+namespace
+{
+
+DbgCtl dbg_ctl_inactivity_cop{"inactivity_cop"};
+DbgCtl dbg_ctl_inactivity_cop_check{"inactivity_cop_check"};
+DbgCtl dbg_ctl_inactivity_cop_verbose{"inactivity_cop_verbose"};
+DbgCtl dbg_ctl_net_queue{"net_queue"};
+DbgCtl dbg_ctl_v_net_queue{"v_net_queue"};
+DbgCtl dbg_ctl_iocore_net_main{"iocore_net_main"};
+
+#ifdef DEBUG
```

Comment:
Is it worth `#ifdef` for these? It's not much of a footprint.

Reply:
I'm indifferent, what's you preference?  One could make the argument that DDbg() itself is not worth it.